src/metabase/api/search.clj

```diff
@@ -172,6 +172,17 @@
                                [:= :r.most_recent true]
                                [:= :r.model (search.config/search-model->revision-model model)]])))
 
+(mu/defn ^:private with-moderated-status :- :map
+  [query :- :map
+   model :- [:enum "card" "dataset"]]
+  (-> query
+      (replace-select :moderated_status [:mr.status])
```

Comment:
Don't you need to do `(replace-select :moderated_status [:mr.status :moderated_status])`? 
If not I'm wondering why L168 is written that way

Reply:
you're right, we need it to.

Reply:
Alternatively (or even better), you could update `replace-select` so that this mistake can never happen again. 

So instead of doing 
`(replace-select :moderated_status [:mr.status :moderated_status])`

you can just call
`(replace-select :moderated_status :mr.status)`

and it will do the same thing.

Reply:
good point, did that in 97dd143db9